src/Module.php

```diff
@@ -18,7 +18,7 @@
 class Module extends BaseModule
 {
     /** @var array<string,string */
-    private const VERSION_OPERATORS = [
+    private static $VERSION_OPERATORS = [
```

Comment:
This should probably be `const VERSION_OPERATORS` (just take out the visibility modifier)

Reply:
/me checks if that runs

Reply:
yup, that works :D